frontend/src/modules/Shares/views/ShareList.js

```diff
@@ -14,9 +14,9 @@ import { Helmet } from 'react-helmet-async';
 import { FiSend } from 'react-icons/fi';
 import { RiInboxArchiveLine } from 'react-icons/ri';
 import { Link as RouterLink } from 'react-router-dom';
-import { ChevronRightIcon, useSettings } from '../../../../design';
+import { ChevronRightIcon, useSettings } from 'design';
 import { ShareInboxList } from 'modules/Shared';
```

Comment:
I understand why we have the Inbox in `modules/Shared` and the outbox in `Shares` but it is a bit confusing as a developer to manage one in one place and the other in another place. I am quite ignorant here, but it is possible or recommended to have them both in `Shares` and import the inbox from `Shares` in the `Dataset` module

Reply:
Following the migration steps [here](https://github.com/awslabs/aws-dataall/issues/398), components or scripts shared among different modules should be under a common directory in this case `src/Shared/Shares`, however, I agree with your point, if we are following logical relationships every related script should be under the same folder `src/Shares`. :)

Reply:
@itsmo-amzn I think for the `Shares` module, we should leave `ShareInboxList` and `ShareInboxListItem` in the `src/modules/Shares/components`, I agree that it is confusing to find `ShareOutboxList` in components and `ShareInboxList` in the global shared folder, what do you think?

Reply:
Good point! But how would this pan out in the whole modularization scheme?
According to this proposed design, we should keep all shared components/services in their original module directory, and in order to access them from other modules, we would simply use absolute addressing `import { ShareInboxList } from 'modules/Shares'`. I believe we discussed the approach for "Shared" modules before. But maybe we can make an exception for this module?

Reply:
I agree, an exception can be made for this, another thing to note is that other components that we put in the Shared folder (`Stack`, `Comments`, `KeyValueTags`) doesn't have a view. So technically they cannot be a module, going by that the `Shares` module is different

Reply:
@dlpzx @itsmo-amzn This has been updated with [1945d73](https://github.com/awslabs/aws-dataall/pull/641/commits/1945d739e402799edb175722b604296b020bb962)